packages/ui/src/views/connectors/connectors-list/connectors-list.tsx

```diff
@@ -76,7 +86,13 @@ export function ConnectorsList({
             >
               <Table.Cell className="max-w-80 content-center truncate">
                 <div className="flex items-center gap-2.5">
-                  <Icon name="connectors" size={24} />
+                  <div className="min-w-[40px]">
+                    {connector.type && LogoNameMap[connector.type.toLowerCase()] ? (
+                      <Logo name={connector.type.toLowerCase()} size={32} />
```

Comment:
i don't think connector type will always map to a logo name.
we should maintain more control over this by maintaining a map of type to logo.

Reply:
Agreed, I have cleaned up this a bit to dynamically generate mapping of logo names to the actual logo imported from `simple-icons` package.